nibe/connection/nibegw.py

```diff
@@ -153,6 +158,62 @@ def datagram_received(self, data, addr):
             elif cmd == "PRODUCT_INFO_MSG":
                 with suppress(InvalidStateError, CancelledError, KeyError):
                     self._futures["product_info"].set_result(msg.fields.value.data)
+            elif cmd == "RMU_DATA_MSG":
+                data = msg.fields.value.data
+                self._on_coil_value(40004, (data.bt1_outdoor_temperature - 5) / 10)
```

Comment:
nibepi does some strange dance that it adds 5 to the value if it's negative. I think it's due to the unsigned parsing in javascript.